tests: "grpc-gateway: grpc: timed out trying to connect"

Binary: cockroach/static-tests.tar.gz sha: https://github.com/cockroachdb/cockroach/commits/371017263b122dbd2423dc1c2c6b8e98b1ff19f0

Stress build found a failed test:

```
=== RUN   TestClientGetAndPutProto
W160414 02:59:08.542315 gossip/gossip.go:887  not connected to cluster; use --join to specify a connected node
I160414 02:59:08.542703 storage/engine/rocksdb.go:137  opening in memory rocksdb instance
I160414 02:59:08.543465 server/node.go:360  store store=0:0 ([]=) not bootstrapped
W160414 02:59:08.544273 gossip/gossip.go:887  not connected to cluster; use --join to specify a connected node
I160414 02:59:08.544450 storage/replica_command.go:1409  range 1: new leader lease replica {1 1 1} 1970-01-01 00:00:00 +0000 UTC 405722h59m9.544195844s
I160414 02:59:08.544746 server/node.go:310  **** cluster {cf021b91-193a-48d9-b3eb-2fc84dcbfae6} has been created
I160414 02:59:08.544762 server/node.go:311  **** add additional nodes by specifying --join=127.0.0.1:41374
W160414 02:59:08.544775 gossip/gossip.go:887  not connected to cluster; use --join to specify a connected node
I160414 02:59:08.547556 server/node.go:373  initialized store store=1:1 ([]=): {Capacity:8312655872 Available:6032011264 RangeCount:0}
I160414 02:59:08.547585 server/node.go:285  node ID 1 initialized
I160414 02:59:08.547630 storage/stores.go:286  read 0 node addresses from persistent storage
I160414 02:59:08.547689 server/node.go:494  connecting to gossip network to verify cluster ID...
I160414 02:59:08.547901 server/node.go:515  node connected via gossip and verified as part of cluster {"cf021b91-193a-48d9-b3eb-2fc84dcbfae6"}
I160414 02:59:08.547940 server/node.go:338  [node=1] Started node with [[]=] engine(s) and attributes []
I160414 02:59:08.548020 server/server.go:363  starting https server at 127.0.0.1:33548
I160414 02:59:08.548038 server/server.go:364  starting grpc/postgres server at 127.0.0.1:41374
I160414 02:59:08.558049 storage/split_queue.go:100  splitting range=1 [/Min-/Max) at keys [/Table/11/0 /Table/12/0 /Table/13/0 /Table/14/0]
I160414 02:59:08.558078 server/updates.go:147  No previous updates check time.
I160414 02:59:08.559217 storage/replica_command.go:1862  initiating a split of range=1 [/Min-/Max) at key /Table/11
I160414 02:59:08.562728 storage/replica_command.go:1409  range 2: new leader lease replica {1 1 1} 1970-01-01 00:00:00 +0000 UTC 405722h59m9.594894629s
I160414 02:59:08.562821 storage/replica_command.go:1862  initiating a split of range=2 [/Table/11-/Max) at key /Table/12
I160414 02:59:09.580794 storage/replica_command.go:1409  range 3: new leader lease replica {1 1 1} 1970-01-01 00:00:00 +0000 UTC 405722h59m10.580794625s
I160414 02:59:09.580794 storage/replica_command.go:1862  initiating a split of range=3 [/Table/12-/Max) at key /Table/13
I160414 02:59:09.580794 gossip/gossip.go:913  starting client to 127.0.0.1:41374
I160414 02:59:09.601432 /go/src/google.golang.org/grpc/server.go:273  grpc: Server.Serve failed to complete security handshake from "127.0.0.1:41716": EOF
I160414 02:59:09.603307 storage/replica_command.go:1409  range 4: new leader lease replica {1 1 1} 1970-01-01 00:00:00 +0000 UTC 405722h59m10.603075152s
I160414 02:59:09.603404 storage/replica_command.go:1862  initiating a split of range=4 [/Table/13-/Max) at key /Table/14
I160414 02:59:09.634642 gossip/client.go:83  closing client to node 1 (127.0.0.1:41374): gossip/client.go:177: stopping outgoing client to node 1 (127.0.0.1:41374); loopback connection
--- FAIL: TestClientGetAndPutProto (6.08s)
	testserver.go:60: Could not start server: server/admin.go:227: error constructing grpc-gateway: grpc: timed out trying to connect. are your certificates valid?
	leaktest.go:82: Leaked goroutine: goroutine 132 [select]:
		github.com/cockroachdb/cockroach/storage.(*baseQueue).processLoop.func1()
			/go/src/github.com/cockroachdb/cockroach/storage/queue.go:356 +0x303
		github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8207de000, 0xc820242000)
			/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:139 +0x52
		created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
			/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:140 +0x62
	leaktest.go:82: Leaked goroutine: goroutine 331 [select]:
		github.com/cockroachdb/cockroach/storage.(*baseQueue).processLoop.func1()
			/go/src/github.com/cockroachdb/cockroach/storage/queue.go:356 +0x303
		github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8207de000, 0xc8204aa1e0)
			/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:139 +0x52
		created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
			/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:140 +0x62
	leaktest.go:82: Leaked goroutine: goroutine 332 [select]:
		github.com/cockroachdb/cockroach/storage.(*replicaScanner).waitAndProcess(0xc820586b60, 0xecea0fe7c, 0x20a9cb38, 0x248ff00, 0xc8205d1cc0, 0xc8207de000, 0xc8208fea80, 0xc820e6fe98)
			/go/src/github.com/cockroachdb/cockroach/storage/scanner.go:162 +0x568
		github.com/cockroachdb/cockroach/storage.(*replicaScanner).scanLoop.func1.1(0xc8208fea80, 0xc8207b1b30)
			/go/src/github.com/cockroachdb/cockroach/storage/scanner.go:204 +0x78
		github.com/cockroachdb/cockroach/storage.(*storeRangeSet).Visit(0xc8207a60c0, 0xc820462b80)
			/go/src/github.com/cockroachdb/cockroach/storage/store.go:237 +0x246
		github.com/cockroachdb/cockroach/storage.(*replicaScanner).scanLoop.func1()
			/go/src/github.com/cockroachdb/cockroach/storage/scanner.go:206 +0x21e
		github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8207de000, 0xc8204aa200)
			/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:139 +0x52
		created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
			/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:140 +0x62
	leaktest.go:82: Leaked goroutine: goroutine 333 [select]:
		github.com/cockroachdb/cockroach/storage.(*baseQueue).processLoop.func1()
			/go/src/github.com/cockroachdb/cockroach/storage/queue.go:356 +0x303
		github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8207de000, 0xc8204aa320)
			/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:139 +0x52
		created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
			/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:140 +0x62
	leaktest.go:82: Leaked goroutine: goroutine 334 [select]:
		github.com/cockroachdb/cockroach/storage.(*baseQueue).processLoop.func1()
			/go/src/github.com/cockroachdb/cockroach/storage/queue.go:356 +0x303
		github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8207de000, 0xc8204aa340)
			/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:139 +0x52
		created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
			/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:140 +0x62
	leaktest.go:82: Leaked goroutine: goroutine 336 [IO wait]:
		net.runtime_pollWait(0x7f2bc9e7f9d0, 0x72, 0xc820675000)
			/usr/local/go/src/runtime/netpoll.go:160 +0x60
		net.(*pollDesc).Wait(0xc8205cf560, 0x72, 0x0, 0x0)
			/usr/local/go/src/net/fd_poll_runtime.go:73 +0x3a
		net.(*pollDesc).WaitRead(0xc8205cf560, 0x0, 0x0)
			/usr/local/go/src/net/fd_poll_runtime.go:78 +0x36
		net.(*netFD).Read(0xc8205cf500, 0xc820675000, 0x1000, 0x1000, 0x0, 0x7f2bc9e6c050, 0xc82000e068)
			/usr/local/go/src/net/fd_unix.go:250 +0x23a
		net.(*conn).Read(0xc82093a010, 0xc820675000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
			/usr/local/go/src/net/net.go:172 +0xe4
		github.com/cockroachdb/cmux.(*MuxConn).Read(0xc8201b4f20, 0xc820675000, 0x1000, 0x1000, 0xd, 0x0, 0x0)
			/go/src/github.com/cockroachdb/cmux/cmux.go:204 +0x117
		crypto/tls.(*block).readFromUntil(0xc8207a76e0, 0x7f2bc9e86e18, 0xc8201b4f20, 0x5, 0x0, 0x0)
			/usr/local/go/src/crypto/tls/conn.go:460 +0xcc
		crypto/tls.(*Conn).readRecord(0xc8208bc300, 0x1a8f117, 0x0, 0x0)
			/usr/local/go/src/crypto/tls/conn.go:562 +0x2d1
		crypto/tls.(*Conn).Read(0xc8208bc300, 0xc820742000, 0x8000, 0x8000, 0x0, 0x0, 0x0)
			/usr/local/go/src/crypto/tls/conn.go:939 +0x167
		bufio.(*Reader).fill(0xc8202341e0)
			/usr/local/go/src/bufio/bufio.go:97 +0x1e9
		bufio.(*Reader).Read(0xc8202341e0, 0xc8200d5ef8, 0x9, 0x9, 0x19, 0x0, 0x0)
			/usr/local/go/src/bufio/bufio.go:207 +0x260
		io.ReadAtLeast(0x7f2bb5fbd180, 0xc8202341e0, 0xc8200d5ef8, 0x9, 0x9, 0x9, 0x0, 0x0, 0x0)
			/usr/local/go/src/io/io.go:297 +0xe6
		io.ReadFull(0x7f2bb5fbd180, 0xc8202341e0, 0xc8200d5ef8, 0x9, 0x9, 0xc8204e1a48, 0x0, 0x0)
			/usr/local/go/src/io/io.go:315 +0x62
		golang.org/x/net/http2.readFrameHeader(0xc8200d5ef8, 0x9, 0x9, 0x7f2bb5fbd180, 0xc8202341e0, 0x0, 0x0, 0x0, 0x0)
			/go/src/golang.org/x/net/http2/frame.go:236 +0xa5
		golang.org/x/net/http2.(*Framer).ReadFrame(0xc8200d5ec0, 0x0, 0x0, 0x0, 0x0)
			/go/src/golang.org/x/net/http2/frame.go:463 +0x106
		google.golang.org/grpc/transport.(*framer).readFrame(0xc8207a79e0, 0x0, 0x0, 0x0, 0x0)
			/go/src/google.golang.org/grpc/transport/http_util.go:406 +0x3d
		google.golang.org/grpc/transport.(*http2Server).HandleStreams(0xc82054c510, 0xc8207a7a70)
			/go/src/google.golang.org/grpc/transport/http2_server.go:247 +0x646
		google.golang.org/grpc.(*Server).serveStreams(0xc82014dd80, 0x7f2bb5fbd1d0, 0xc82054c510)
			/go/src/google.golang.org/grpc/server.go:325 +0x159
		google.golang.org/grpc.(*Server).serveNewHTTP2Transport(0xc82014dd80, 0x7f2bb5fbd0a8, 0xc8208bc300, 0x7f2bb5fbd108, 0xc8205d7c30)
			/go/src/google.golang.org/grpc/server.go:312 +0x49d
		google.golang.org/grpc.(*Server).handleRawConn(0xc82014dd80, 0x7f2bc9e86db8, 0xc8201b4f20)
			/go/src/google.golang.org/grpc/server.go:289 +0x4ee
		created by google.golang.org/grpc.(*Server).Serve
			/go/src/google.golang.org/grpc/server.go:261 +0x372
	leaktest.go:82: Leaked goroutine: goroutine 380 [select]:
		github.com/cockroachdb/cockroach/storage.(*baseQueue).processLoop.func1()
			/go/src/github.com/cockroachdb/cockroach/storage/queue.go:356 +0x303
		github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8207de000, 0xc820420120)
			/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:139 +0x52
		created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
			/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:140 +0x62
	leaktest.go:82: Leaked goroutine: goroutine 409 [select]:
		github.com/cockroachdb/cockroach/storage.(*Store).processRaft.func1()
			/go/src/github.com/cockroachdb/cockroach/storage/store.go:1877 +0xbee
		github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8207de000, 0xc820564100)
			/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:139 +0x52
		created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
			/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:140 +0x62
	leaktest.go:82: Leaked goroutine: goroutine 410 [select]:
		github.com/cockroachdb/cockroach/storage.(*Store).Start.func3()
			/go/src/github.com/cockroachdb/cockroach/storage/store.go:794 +0x13d
		github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8207de000, 0xc8204ae080)
			/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:139 +0x52
		created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
			/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:140 +0x62
	leaktest.go:82: Leaked goroutine: goroutine 411 [select]:
		github.com/cockroachdb/cockroach/storage.(*Store).startGossip.func1()
			/go/src/github.com/cockroachdb/cockroach/storage/store.go:865 +0x3cf
		github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8207de000, 0xc8204ae0a0)
			/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:139 +0x52
		created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
			/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:140 +0x62
	leaktest.go:82: Leaked goroutine: goroutine 412 [select]:
		github.com/cockroachdb/cockroach/storage.(*Store).startGossip.func2()
			/go/src/github.com/cockroachdb/cockroach/storage/store.go:884 +0x3cf
		github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8207de000, 0xc8204ae0c0)
			/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:139 +0x52
		created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
			/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:140 +0x62
	leaktest.go:82: Leaked goroutine: goroutine 416 [IO wait]:
		net.runtime_pollWait(0x7f2bc9e80450, 0x72, 0x0)
			/usr/local/go/src/runtime/netpoll.go:160 +0x60
		net.(*pollDesc).Wait(0xc8207092c0, 0x72, 0x0, 0x0)
			/usr/local/go/src/net/fd_poll_runtime.go:73 +0x3a
		net.(*pollDesc).WaitRead(0xc8207092c0, 0x0, 0x0)
			/usr/local/go/src/net/fd_poll_runtime.go:78 +0x36
		net.(*netFD).accept(0xc820709260, 0x0, 0x7f2bc9e85a80, 0xc82000a260)
			/usr/local/go/src/net/fd_unix.go:426 +0x27c
		net.(*TCPListener).AcceptTCP(0xc82021a0f0, 0xc8207b7ed8, 0x0, 0x0)
			/usr/local/go/src/net/tcpsock_posix.go:254 +0x4d
		net.(*TCPListener).Accept(0xc82021a0f0, 0x0, 0x0, 0x0, 0x0)
			/usr/local/go/src/net/tcpsock_posix.go:264 +0x3d
		github.com/cockroachdb/cmux.(*cMux).Serve(0xc8207a0b40, 0x0, 0x0)
			/go/src/github.com/cockroachdb/cmux/cmux.go:110 +0x9d
		github.com/cockroachdb/cockroach/server.(*Server).Start.func9()
			/go/src/github.com/cockroachdb/cockroach/server/server.go:370 +0x32
		github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8207de000, 0xc8204ae260)
			/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:139 +0x52
		created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
			/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:140 +0x62
	leaktest.go:82: Leaked goroutine: goroutine 417 [select]:
		github.com/cockroachdb/cockroach/storage.(*baseQueue).processLoop.func1()
			/go/src/github.com/cockroachdb/cockroach/storage/queue.go:356 +0x303
		github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8207de000, 0xc8204ae320)
			/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:139 +0x52
		created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
			/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:140 +0x62
	leaktest.go:82: Leaked goroutine: goroutine 425 [select]:
		github.com/cockroachdb/cockroach/server.(*Node).startComputePeriodicMetrics.func1()
			/go/src/github.com/cockroachdb/cockroach/server/node.go:561 +0x23b
		github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8207de000, 0xc820257100)
			/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:139 +0x52
		created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
			/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:140 +0x62
	leaktest.go:82: Leaked goroutine: goroutine 426 [select]:
		github.com/cockroachdb/cockroach/server.(*Node).startGossip.func1()
			/go/src/github.com/cockroachdb/cockroach/server/node.go:528 +0x3e3
		github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8207de000, 0xc820257140)
			/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:139 +0x52
		created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
			/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:140 +0x62
	leaktest.go:82: Leaked goroutine: goroutine 428 [select]:
		github.com/cockroachdb/cockroach/server.(*Server).startSampleEnvironment.func1()
			/go/src/github.com/cockroachdb/cockroach/server/server.go:387 +0x15d
		github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8207de000, 0xc8202573a0)
			/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:139 +0x52
		created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
			/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:140 +0x62
	leaktest.go:82: Leaked goroutine: goroutine 429 [select]:
		github.com/cockroachdb/cockroach/ts.(*poller).start.func1()
			/go/src/github.com/cockroachdb/cockroach/ts/db.go:78 +0x150
		github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8207de000, 0xc82004c190)
			/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:139 +0x52
		created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
			/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:140 +0x62
	leaktest.go:82: Leaked goroutine: goroutine 430 [select]:
		github.com/cockroachdb/cockroach/server.(*Node).startWriteSummaries.func1()
			/go/src/github.com/cockroachdb/cockroach/server/node.go:595 +0x35c
		github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8207de000, 0xc8202573e0)
			/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:139 +0x52
		created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
			/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:140 +0x62
	leaktest.go:82: Leaked goroutine: goroutine 431 [select]:
		github.com/cockroachdb/cockroach/sql.(*SchemaChangeManager).Start.func1()
			/go/src/github.com/cockroachdb/cockroach/sql/schema_changer.go:450 +0x16f8
		github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8207de000, 0xc820257420)
			/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:139 +0x52
		created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
			/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:140 +0x62
	leaktest.go:82: Leaked goroutine: goroutine 432 [select]:
		github.com/cockroachdb/cockroach/server.(*Server).periodicallyCheckForUpdates.func1()
			/go/src/github.com/cockroachdb/cockroach/server/updates.go:99 +0x1e5
		github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8207de000, 0xc82004c1a0)
			/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:139 +0x52
		created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
			/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:140 +0x62
	leaktest.go:82: Leaked goroutine: goroutine 433 [select]:
		google.golang.org/grpc.(*Conn).transportMonitor(0xc82040c0e0)
			/go/src/google.golang.org/grpc/clientconn.go:510 +0x1d3
		google.golang.org/grpc.NewConn.func1(0xc82040c0e0)
			/go/src/google.golang.org/grpc/clientconn.go:321 +0x1b5
		created by google.golang.org/grpc.NewConn
			/go/src/google.golang.org/grpc/clientconn.go:322 +0x4dd
	leaktest.go:82: Leaked goroutine: goroutine 445 [select]:
		github.com/cockroachdb/cockroach/storage.(*baseQueue).processLoop.func1()
			/go/src/github.com/cockroachdb/cockroach/storage/queue.go:356 +0x303
		github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8207de000, 0xc82018d860)
			/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:139 +0x52
		created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
			/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:140 +0x62
	leaktest.go:82: Leaked goroutine: goroutine 446 [select]:
		google.golang.org/grpc/transport.(*http2Client).controller(0xc820930000)
			/go/src/google.golang.org/grpc/transport/http2_client.go:835 +0x5da
		created by google.golang.org/grpc/transport.newHTTP2Client
			/go/src/google.golang.org/grpc/transport/http2_client.go:194 +0x153b
	leaktest.go:82: Leaked goroutine: goroutine 447 [IO wait]:
		net.runtime_pollWait(0x7f2bc9e7f6d0, 0x72, 0xc820134400)
			/usr/local/go/src/runtime/netpoll.go:160 +0x60
		net.(*pollDesc).Wait(0xc820056140, 0x72, 0x0, 0x0)
			/usr/local/go/src/net/fd_poll_runtime.go:73 +0x3a
		net.(*pollDesc).WaitRead(0xc820056140, 0x0, 0x0)
			/usr/local/go/src/net/fd_poll_runtime.go:78 +0x36
		net.(*netFD).Read(0xc8200560e0, 0xc820134400, 0x400, 0x400, 0x0, 0x7f2bc9e6c050, 0xc82000e068)
			/usr/local/go/src/net/fd_unix.go:250 +0x23a
		net.(*conn).Read(0xc82002a028, 0xc820134400, 0x400, 0x400, 0x0, 0x0, 0x0)
			/usr/local/go/src/net/net.go:172 +0xe4
		crypto/tls.(*block).readFromUntil(0xc8207a6bd0, 0x7f2bc9e86c40, 0xc82002a028, 0x5, 0x0, 0x0)
			/usr/local/go/src/crypto/tls/conn.go:460 +0xcc
		crypto/tls.(*Conn).readRecord(0xc8208bc000, 0x1a8f117, 0x0, 0x0)
			/usr/local/go/src/crypto/tls/conn.go:562 +0x2d1
		crypto/tls.(*Conn).Read(0xc8208bc000, 0xc820464000, 0x8000, 0x8000, 0x0, 0x0, 0x0)
			/usr/local/go/src/crypto/tls/conn.go:939 +0x167
		bufio.(*Reader).fill(0xc820720540)
			/usr/local/go/src/bufio/bufio.go:97 +0x1e9
		bufio.(*Reader).Read(0xc820720540, 0xc8205ac038, 0x9, 0x9, 0xc81ffbc0e6, 0x0, 0x0)
			/usr/local/go/src/bufio/bufio.go:207 +0x260
		io.ReadAtLeast(0x7f2bb5fbd180, 0xc820720540, 0xc8205ac038, 0x9, 0x9, 0x9, 0x0, 0x0, 0x0)
			/usr/local/go/src/io/io.go:297 +0xe6
		io.ReadFull(0x7f2bb5fbd180, 0xc820720540, 0xc8205ac038, 0x9, 0x9, 0xc82087e0c8, 0x0, 0x0)
			/usr/local/go/src/io/io.go:315 +0x62
		golang.org/x/net/http2.readFrameHeader(0xc8205ac038, 0x9, 0x9, 0x7f2bb5fbd180, 0xc820720540, 0x0, 0xc800000000, 0x0, 0x0)
			/go/src/golang.org/x/net/http2/frame.go:236 +0xa5
		golang.org/x/net/http2.(*Framer).ReadFrame(0xc8205ac000, 0x0, 0x0, 0x0, 0x0)
			/go/src/golang.org/x/net/http2/frame.go:463 +0x106
		google.golang.org/grpc/transport.(*framer).readFrame(0xc8206a0270, 0x0, 0x0, 0x0, 0x0)
			/go/src/google.golang.org/grpc/transport/http_util.go:406 +0x3d
		google.golang.org/grpc/transport.(*http2Client).reader(0xc820930000)
			/go/src/google.golang.org/grpc/transport/http2_client.go:757 +0x109
		created by google.golang.org/grpc/transport.newHTTP2Client
			/go/src/google.golang.org/grpc/transport/http2_client.go:200 +0x159a
	leaktest.go:82: Leaked goroutine: goroutine 457 [chan receive]:
		github.com/cockroachdb/cockroach/rpc.NewContext.func1()
			/go/src/github.com/cockroachdb/cockroach/rpc/context.go:89 +0x57
		github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8207de000, 0xc8207e2960)
			/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:139 +0x52
		created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
			/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:140 +0x62
	leaktest.go:82: Leaked goroutine: goroutine 458 [select]:
		github.com/cockroachdb/cockroach/storage.(*StorePool).start.func1()
			/go/src/github.com/cockroachdb/cockroach/storage/store_pool.go:219 +0x23f
		github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8207de000, 0xc8207e2ae0)
			/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:139 +0x52
		created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
			/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:140 +0x62
	leaktest.go:82: Leaked goroutine: goroutine 459 [select]:
		github.com/cockroachdb/cockroach/kv.(*TxnCoordSender).startStats(0xc82085b140)
			/go/src/github.com/cockroachdb/cockroach/kv/txn_coord_sender.go:239 +0xd21
		github.com/cockroachdb/cockroach/kv.(*TxnCoordSender).(github.com/cockroachdb/cockroach/kv.startStats)-fm()
			/go/src/github.com/cockroachdb/cockroach/kv/txn_coord_sender.go:224 +0x20
		github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8207de000, 0xc82086eff0)
			/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:139 +0x52
		created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
			/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:140 +0x62
	leaktest.go:82: Leaked goroutine: goroutine 460 [select]:
		github.com/cockroachdb/cockroach/sql.(*LeaseManager).RefreshLeases.func1()
			/go/src/github.com/cockroachdb/cockroach/sql/lease.go:613 +0xde4
		github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8207de000, 0xc8208c5320)
			/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:139 +0x52
		created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
			/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:140 +0x62
	leaktest.go:82: Leaked goroutine: goroutine 461 [select]:
		github.com/cockroachdb/cockroach/sql.NewExecutor.func1()
			/go/src/github.com/cockroachdb/cockroach/sql/executor.go:207 +0x15c
		github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8207de000, 0xc8202725f0)
			/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:139 +0x52
		created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
			/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:140 +0x62
	leaktest.go:82: Leaked goroutine: goroutine 462 [chan receive]:
		github.com/cockroachdb/cockroach/storage/engine.(*RocksDB).Open.func1(0xc82085bf20)
			/go/src/github.com/cockroachdb/cockroach/storage/engine/rocksdb.go:164 +0x3a
		created by github.com/cockroachdb/cockroach/storage/engine.(*RocksDB).Open
			/go/src/github.com/cockroachdb/cockroach/storage/engine/rocksdb.go:165 +0x8f4
	leaktest.go:82: Leaked goroutine: goroutine 463 [chan receive]:
		github.com/cockroachdb/cockroach/server.(*Server).Start.func1()
			/go/src/github.com/cockroachdb/cockroach/server/server.go:259 +0x62
		github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8207de000, 0xc8207e2f20)
			/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:139 +0x52
		created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
			/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:140 +0x62
	leaktest.go:82: Leaked goroutine: goroutine 464 [chan receive]:
		github.com/cockroachdb/cockroach/server.(*Server).Start.func2()
			/go/src/github.com/cockroachdb/cockroach/server/server.go:280 +0x59
		github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8207de000, 0xc8207e3000)
			/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:139 +0x52
		created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
			/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:140 +0x62
	leaktest.go:82: Leaked goroutine: goroutine 465 [IO wait]:
		net.runtime_pollWait(0x7f2bc9e80390, 0x72, 0x0)
			/usr/local/go/src/runtime/netpoll.go:160 +0x60
		net.(*pollDesc).Wait(0xc820709330, 0x72, 0x0, 0x0)
			/usr/local/go/src/net/fd_poll_runtime.go:73 +0x3a
		net.(*pollDesc).WaitRead(0xc820709330, 0x0, 0x0)
			/usr/local/go/src/net/fd_poll_runtime.go:78 +0x36
		net.(*netFD).accept(0xc8207092d0, 0x0, 0x7f2bc9e85a80, 0xc8204cac00)
			/usr/local/go/src/net/fd_unix.go:426 +0x27c
		net.(*TCPListener).AcceptTCP(0xc82021a108, 0x621030, 0x0, 0x0)
			/usr/local/go/src/net/tcpsock_posix.go:254 +0x4d
		net.(*TCPListener).Accept(0xc82021a108, 0x0, 0x0, 0x0, 0x0)
			/usr/local/go/src/net/tcpsock_posix.go:264 +0x3d
		github.com/cockroachdb/cmux.(*cMux).Serve(0xc8207a0b80, 0x0, 0x0)
			/go/src/github.com/cockroachdb/cmux/cmux.go:110 +0x9d
		github.com/cockroachdb/cockroach/server.(*Server).Start.func3()
			/go/src/github.com/cockroachdb/cockroach/server/server.go:292 +0x32
		github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8207de000, 0xc8207e3220)
			/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:139 +0x52
		created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
			/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:140 +0x62
	leaktest.go:82: Leaked goroutine: goroutine 466 [chan receive]:
		github.com/cockroachdb/cmux.(*muxListener).Accept(0xc8207e31e0, 0x0, 0x0, 0x0, 0x0)
			<autogenerated>:9 +0x125
		net/http.(*Server).Serve(0xc82014de80, 0x7f2bc9e806c8, 0xc8207e31e0, 0x0, 0x0)
			/usr/local/go/src/net/http/server.go:2117 +0x129
		github.com/cockroachdb/cockroach/util.ServeHandler.func2()
			/go/src/github.com/cockroachdb/cockroach/util/net.go:90 +0x66
		github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8207de000, 0xc8207a0bc0)
			/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:139 +0x52
		created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
			/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:140 +0x62
	leaktest.go:82: Leaked goroutine: goroutine 467 [chan receive]:
		github.com/cockroachdb/cmux.(*muxListener).Accept(0xc8207e3200, 0x0, 0x0, 0x0, 0x0)
			<autogenerated>:9 +0x125
		crypto/tls.(*listener).Accept(0xc8207e32c0, 0x0, 0x0, 0x0, 0x0)
			/usr/local/go/src/crypto/tls/tls.go:52 +0x60
		net/http.(*Server).Serve(0xc82014df00, 0x7f2bc9e84700, 0xc8207e32c0, 0x0, 0x0)
			/usr/local/go/src/net/http/server.go:2117 +0x129
		github.com/cockroachdb/cockroach/util.ServeHandler.func2()
			/go/src/github.com/cockroachdb/cockroach/util/net.go:90 +0x66
		github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8207de000, 0xc8207a0cc0)
			/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:139 +0x52
		created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
			/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:140 +0x62
	leaktest.go:82: Leaked goroutine: goroutine 468 [chan receive]:
		github.com/cockroachdb/cmux.(*muxListener).Accept(0xc8207e2f60, 0x0, 0x0, 0x0, 0x0)
			<autogenerated>:9 +0x125
		google.golang.org/grpc.(*Server).Serve(0xc82014dd80, 0x7f2bc9e806c8, 0xc8207e2f60, 0x0, 0x0)
			/go/src/google.golang.org/grpc/server.go:252 +0x1b5
		github.com/cockroachdb/cockroach/server.(*Server).Start.func5()
			/go/src/github.com/cockroachdb/cockroach/server/server.go:306 +0x43
		github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8207de000, 0xc8207e3340)
			/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:139 +0x52
		created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
			/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:140 +0x62
	leaktest.go:82: Leaked goroutine: goroutine 469 [chan receive]:
		github.com/cockroachdb/cmux.(*muxListener).Accept(0xc8207e2f40, 0x0, 0x0, 0x0, 0x0)
			<autogenerated>:9 +0x125
		github.com/cockroachdb/cockroach/util.ServeHandler.func3(0x7f2bc9e806c8, 0xc8207e2f40, 0xc8206e5bc0, 0x0, 0x0)
			/go/src/github.com/cockroachdb/cockroach/util/net.go:105 +0x66
		github.com/cockroachdb/cockroach/server.(*Server).Start.func6()
			/go/src/github.com/cockroachdb/cockroach/server/server.go:314 +0x96
		github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8207de000, 0xc8208c5e90)
			/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:139 +0x52
		created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
			/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:140 +0x62
	leaktest.go:82: Leaked goroutine: goroutine 470 [chan receive]:
		github.com/cockroachdb/cockroach/gossip.(*server).start.func3()
			/go/src/github.com/cockroachdb/cockroach/gossip/server.go:298 +0x5c
		github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8207de000, 0xc8207e33c0)
			/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:139 +0x52
		created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
			/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:140 +0x62
	leaktest.go:82: Leaked goroutine: goroutine 471 [select]:
		github.com/cockroachdb/cockroach/gossip.(*Gossip).bootstrap.func1()
			/go/src/github.com/cockroachdb/cockroach/gossip/gossip.go:757 +0x219
		github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8207de000, 0xc8207e33e0)
			/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:139 +0x52
		created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
			/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:140 +0x62
	leaktest.go:82: Leaked goroutine: goroutine 472 [select]:
		github.com/cockroachdb/cockroach/gossip.(*Gossip).manage.func1()
			/go/src/github.com/cockroachdb/cockroach/gossip/gossip.go:786 +0x33b
		github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8207de000, 0xc8207e3400)
			/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:139 +0x52
		created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
			/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:140 +0x62
	leaktest.go:82: Leaked goroutine: goroutine 476 [select]:
		github.com/cockroachdb/cockroach/storage.(*replicaScanner).waitAndProcess(0xc8205868c0, 0xecea0fe81, 0x2151e749, 0x248ff00, 0xc8205d1cc0, 0xc8207de000, 0xc8208fea80, 0xc8204dde68)
			/go/src/github.com/cockroachdb/cockroach/storage/scanner.go:162 +0x568
		github.com/cockroachdb/cockroach/storage.(*replicaScanner).scanLoop.func1.1(0xc8208fea80, 0xc8207b1b30)
			/go/src/github.com/cockroachdb/cockroach/storage/scanner.go:204 +0x78
		github.com/cockroachdb/cockroach/storage.(*storeRangeSet).Visit(0xc8207b1c50, 0xc8207e00c0)
			/go/src/github.com/cockroachdb/cockroach/storage/store.go:237 +0x246
		github.com/cockroachdb/cockroach/storage.(*replicaScanner).scanLoop.func1()
			/go/src/github.com/cockroachdb/cockroach/storage/scanner.go:206 +0x21e
		github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8207de000, 0xc8202427a0)
			/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:139 +0x52
		created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
			/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:140 +0x62
	leaktest.go:82: Leaked goroutine: goroutine 478 [select]:
		google.golang.org/grpc/transport.(*http2Server).controller(0xc82054c510)
			/go/src/google.golang.org/grpc/transport/http2_server.go:620 +0x5da
		created by google.golang.org/grpc/transport.newHTTP2Server
			/go/src/google.golang.org/grpc/transport/http2_server.go:134 +0x853
	leaktest.go:82: Leaked goroutine: goroutine 482 [select]:
		github.com/cockroachdb/cockroach/rpc.(*Context).runHeartbeat(0xc820708700, 0xc820096000, 0xc8205642a0, 0xf, 0x0, 0x0)
			/go/src/github.com/cockroachdb/cockroach/rpc/context.go:209 +0x5e5
		github.com/cockroachdb/cockroach/rpc.(*Context).GRPCDial.func1()
			/go/src/github.com/cockroachdb/cockroach/rpc/context.go:157 +0x66
		github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8207de000, 0xc820424360)
			/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:139 +0x52
		created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
			/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:140 +0x62
	leaktest.go:82: Leaked goroutine: goroutine 486 [select]:
		github.com/cockroachdb/cockroach/storage.(*idAllocator).start.func1()
			/go/src/github.com/cockroachdb/cockroach/storage/id_alloc.go:131 +0x910
		github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8207de000, 0xc82052b2b0)
			/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:139 +0x52
		created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
			/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:140 +0x62
```

Run Details:

```
7 runs so far, 0 failures, over 5s
16 runs so far, 0 failures, over 10s
29 runs so far, 0 failures, over 15s
37 runs so far, 0 failures, over 20s
48 runs so far, 0 failures, over 25s
59 runs so far, 0 failures, over 30s
68 runs so far, 0 failures, over 35s
82 runs so far, 0 failures, over 40s
89 runs so far, 0 failures, over 45s
102 runs so far, 0 failures, over 50s
110 runs so far, 0 failures, over 55s
123 runs so far, 0 failures, over 1m0s
131 runs so far, 0 failures, over 1m5s
144 runs so far, 0 failures, over 1m10s
151 runs so far, 0 failures, over 1m15s
163 runs so far, 0 failures, over 1m20s
173 runs so far, 0 failures, over 1m25s
184 runs so far, 0 failures, over 1m30s
195 runs so far, 0 failures, over 1m35s
206 runs so far, 0 failures, over 1m40s
216 runs so far, 0 failures, over 1m45s
228 runs so far, 0 failures, over 1m50s
237 runs so far, 0 failures, over 1m55s
248 runs so far, 0 failures, over 2m0s
260 runs so far, 0 failures, over 2m5s
270 runs so far, 0 failures, over 2m10s
280 runs so far, 0 failures, over 2m15s
291 runs so far, 0 failures, over 2m20s
302 runs so far, 0 failures, over 2m25s
312 runs so far, 0 failures, over 2m30s
323 runs so far, 0 failures, over 2m35s
333 runs so far, 0 failures, over 2m40s
345 runs so far, 0 failures, over 2m45s
354 runs so far, 0 failures, over 2m50s
365 runs so far, 0 failures, over 2m55s
375 runs so far, 0 failures, over 3m0s
385 runs so far, 0 failures, over 3m5s
396 runs so far, 0 failures, over 3m10s
408 runs so far, 0 failures, over 3m15s
417 runs so far, 0 failures, over 3m20s
429 runs so far, 0 failures, over 3m25s
439 runs so far, 0 failures, over 3m30s
450 runs so far, 0 failures, over 3m35s
458 runs so far, 0 failures, over 3m40s
473 runs so far, 0 failures, over 3m45s
481 runs so far, 0 failures, over 3m50s
494 runs so far, 0 failures, over 3m55s
502 runs so far, 0 failures, over 4m0s
514 runs so far, 0 failures, over 4m5s
524 runs so far, 0 failures, over 4m10s
535 runs so far, 0 failures, over 4m15s
546 runs so far, 0 failures, over 4m20s
556 runs so far, 0 failures, over 4m25s
568 runs so far, 0 failures, over 4m30s
579 runs so far, 0 failures, over 4m35s
589 runs so far, 0 failures, over 4m40s
600 runs so far, 0 failures, over 4m45s
610 runs so far, 0 failures, over 4m50s
620 runs so far, 0 failures, over 4m55s
632 runs so far, 0 failures, over 5m0s
643 runs so far, 0 failures, over 5m5s
653 runs so far, 0 failures, over 5m10s
665 runs so far, 0 failures, over 5m15s
675 runs so far, 0 failures, over 5m20s
686 runs so far, 0 failures, over 5m25s
696 runs so far, 0 failures, over 5m30s
707 runs so far, 0 failures, over 5m35s
719 runs so far, 0 failures, over 5m40s
728 runs so far, 0 failures, over 5m45s
739 runs so far, 0 failures, over 5m50s
750 runs so far, 0 failures, over 5m55s
760 runs so far, 0 failures, over 6m0s
772 runs so far, 0 failures, over 6m5s
780 runs so far, 0 failures, over 6m10s
793 runs so far, 0 failures, over 6m15s
804 runs so far, 0 failures, over 6m20s
814 runs so far, 0 failures, over 6m25s
825 runs so far, 0 failures, over 6m30s
836 runs so far, 0 failures, over 6m35s
846 runs so far, 0 failures, over 6m40s
855 runs so far, 0 failures, over 6m45s
867 runs so far, 0 failures, over 6m50s
876 runs so far, 0 failures, over 6m55s
888 runs so far, 0 failures, over 7m0s
897 runs so far, 0 failures, over 7m5s
909 runs so far, 0 failures, over 7m10s
920 runs so far, 0 failures, over 7m15s
930 runs so far, 0 failures, over 7m20s
943 runs so far, 0 failures, over 7m25s
951 runs so far, 0 failures, over 7m30s
963 runs so far, 0 failures, over 7m35s
973 runs so far, 0 failures, over 7m40s
984 runs so far, 0 failures, over 7m45s
994 runs so far, 0 failures, over 7m50s
1006 runs so far, 0 failures, over 7m55s
1015 runs so far, 0 failures, over 8m0s
1026 runs so far, 0 failures, over 8m5s
1035 runs so far, 0 failures, over 8m10s
1047 runs so far, 0 failures, over 8m15s
1057 runs so far, 0 failures, over 8m20s
1066 runs so far, 0 failures, over 8m25s
1079 runs so far, 0 failures, over 8m30s
1089 runs so far, 0 failures, over 8m35s
1099 runs so far, 0 failures, over 8m40s
1110 runs so far, 0 failures, over 8m45s
1121 runs so far, 0 failures, over 8m50s
1132 runs so far, 0 failures, over 8m55s
1142 runs so far, 0 failures, over 9m0s
1152 runs so far, 0 failures, over 9m5s
1165 runs so far, 0 failures, over 9m10s
1174 runs so far, 0 failures, over 9m15s
1186 runs so far, 0 failures, over 9m20s
1196 runs so far, 0 failures, over 9m25s
1205 runs so far, 0 failures, over 9m30s
1219 runs so far, 0 failures, over 9m35s
1227 runs so far, 0 failures, over 9m40s
1240 runs so far, 0 failures, over 9m45s
1251 runs so far, 0 failures, over 9m50s
1261 runs so far, 0 failures, over 9m55s
1272 runs so far, 0 failures, over 10m0s
1283 runs so far, 0 failures, over 10m5s
1292 runs so far, 0 failures, over 10m10s
1304 runs so far, 0 failures, over 10m15s
1314 runs so far, 0 failures, over 10m20s
1325 runs so far, 0 failures, over 10m25s
1335 runs so far, 0 failures, over 10m30s
1346 runs so far, 0 failures, over 10m35s
1356 runs so far, 0 failures, over 10m40s
1368 runs so far, 0 failures, over 10m45s
1378 runs so far, 0 failures, over 10m50s
1388 runs so far, 0 failures, over 10m55s
1400 runs so far, 0 failures, over 11m0s
1411 runs so far, 0 failures, over 11m5s
1419 runs so far, 0 failures, over 11m10s
1430 runs so far, 0 failures, over 11m15s
1439 runs so far, 0 failures, over 11m20s
1443 runs completed, 1 failures, over 11m21s
FAIL
```
Please assign, take a look and update the issue accordingly.

--- a/server/admin.go
+++ b/server/admin.go
@@ -41,6 +41,7 @@ import (
 	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/credentials"
 
+	"github.com/cockroachdb/cockroach/base"
 	"github.com/cockroachdb/cockroach/client"
 	"github.com/cockroachdb/cockroach/keys"
 	"github.com/cockroachdb/cockroach/kv"
@@ -218,7 +219,7 @@ func (s *adminServer) RegisterGRPCGateway(serverCtx *Context) error {
 			// TODO(tamird): remove this timeout. It is currently necessary because
 			// GRPC will not actually bail on a bad certificate error - it will just
 			// retry indefinitely. See https://github.com/grpc/grpc-go/issues/622.
-			grpc.WithTimeout(time.Second),
+			grpc.WithTimeout(base.NetworkTimeout),
 			grpc.WithBlock(),
 			grpc.WithTransportCredentials(credentials.NewTLS(tlsConfig)),
 		)

